pkg/workflows/interfaces/interfaces.go

```diff
@@ -76,6 +76,7 @@ type EksdUpgrader interface {
 
 type PackageInstaller interface {
 	InstallCuratedPackages(ctx context.Context)
+	UpgradeCuratedPackages(ctx context.Context)
```

Comment:
could we have a different interface? or change the name of the interface
I prefer a different interface, but up to you

Reply:
i changed the name :)